src/contracts/interfaces/IVoteWeigher.sol

```diff
@@ -1,25 +1,95 @@
+    function delegation() external view returns (IDelegationManager);
+    /// @notice Returns the service manager contract.
+    function serviceManager() external view returns (IServiceManager);
+
     /**
      * @notice This function computes the total weight of the @param operator in the quorum @param quorumNumber.
      * @dev returns zero in the case that `quorumNumber` is greater than or equal to `NUMBER_OF_QUORUMS`
      */
-    function weightOfOperator(address operator, uint8 quorumNumber) external returns (uint96);
+    function weightOfOperator(uint8 quorumNumber, address operator) external returns (uint96);
 
     /// @notice Number of quorums that are being used by the middleware.
-    function quorumCount() external view returns (uint8);
+    function quorumCount() external view returns (uint16);
 
     /**
      * @notice This defines the earnings split between different quorums. Mapping is quorumNumber => BIPS which the quorum earns, out of the total earnings.
      * @dev The sum of all entries, i.e. sum(quorumBips[0] through quorumBips[NUMBER_OF_QUORUMS - 1]) should *always* be 10,000!
      */
     function quorumBips(uint8 quorumNumber) external view returns (uint256);
+
+    /// @notice Returns the strategy and weight multiplier for the `index`'th strategy in the quorum `quorumNumber`
+    function strategyAndWeightingMultiplierForQuorumByIndex(uint8 quorumNumber, uint256 index) external view returns (StrategyAndWeightingMultiplier memory);
+
+    /// @notice Create a new quorum and add the strategies and their associated weights to the quorum.
+    function createQuorum(
+        StrategyAndWeightingMultiplier[] memory _strategiesConsideredAndMultipliers
+    ) external;
+
+    /// @notice Adds new strategies and the associated multipliers to the @param quorumNumber.
+    function addStrategiesConsideredAndMultipliers(
+        uint8 quorumNumber,
+        StrategyAndWeightingMultiplier[] memory _newStrategiesConsideredAndMultipliers
+    ) external;
+
+    /**
+     * @notice This function is used for removing strategies and their associated weights from the
+     * mapping strategiesConsideredAndMultipliers for a specific @param quorumNumber.
+     * @dev higher indices should be *first* in the list of @param indicesToRemove, since otherwise
+     * the removal of lower index entries will cause a shift in the indices of the other strategiesToRemove
+     */
+    function removeStrategiesConsideredAndMultipliers(
+        uint8 quorumNumber,
+        uint256[] calldata indicesToRemove
+    ) external;
+
+    /**
+     * @notice This function is used for modifying the weights of strategies that are already in the
+     * mapping strategiesConsideredAndMultipliers for a specific @param quorumNumber.
+     * @param strategyIndices is a correctness-check input -- the supplied values must match the indices of the
+     * strategiesToModifyWeightsOf in strategiesConsideredAndMultipliers[quorumNumber]
+     */
```

Comment:
this comment is outdated and incorrect given the new usage.
IMO having a correctness-check still might be a good call.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/76/commits/8c37e88fcacdbfffcddc8953a38292171928ea99 fixed comment

Are you adamant about correctness check?